src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/resources/gtk.properties

```diff
@@ -59,12 +59,12 @@ FileChooser.newFolderNoDirectoryErrorTitle.textAndMnemonic=Error
 FileChooser.newFolderNoDirectoryError.textAndMnemonic=Error creating directory "{0}": No such file or directory
 FileChooser.deleteFileButton.textAndMnemonic=De&lete File
 FileChooser.renameFileButton.textAndMnemonic=&Rename File
-FileChooser.cancelButton.textAndMnemonic=Cancel
-FileChooser.saveButton.textAndMnemonic=Save
-FileChooser.openButton.textAndMnemonic=Open
+FileChooser.cancelButton.textAndMnemonic=&Cancel
+FileChooser.saveButton.textAndMnemonic=&Save
+FileChooser.openButton.textAndMnemonic=&Open
 FileChooser.saveDialogTitle.textAndMnemonic=Save
 FileChooser.openDialogTitle.textAndMnemonic=Open
-FileChooser.pathLabel.textAndMnemonic=&Selection:
+FileChooser.pathLabel.textAndMnemonic=S&election:
```

Comment:
is this a typo? why is the S moved before the &?

Reply:
No, that's not a typo. Proposed fix is to have mnemonic &S for save &e for Selection. 
Refer https://github.com/openjdk/jdk/pull/20612#discussion_r1720283148